js/packages/cli/src/commands/upload.ts

```diff
@@ -117,84 +228,60 @@ export async function uploadV2({
 
   const dedupedAssetKeys = getAssetKeysNeedingUpload(cacheContent.items, files);
   const dirname = path.dirname(files[0]);
+
   let candyMachine = cacheContent.program.candyMachine
     ? new PublicKey(cacheContent.program.candyMachine)
     : undefined;
 
   if (!cacheContent.program.uuid) {
     const firstAssetManifest = getAssetManifest(dirname, '0');
+    if (
+      !firstAssetManifest.properties?.creators?.every(
+        creator => creator.address !== undefined,
+      )
+    ) {
+      throw new Error('Creator address is missing');
+    }
 
     try {
-      const remainingAccounts = [];
-
-      if (splToken) {
```

Comment:
This code wasn't actually doing anything far as I can tell. Remaining Accounts wasn't used after it was populated.